configure.py

```diff
@@ -319,6 +325,24 @@ def get_var(environ_cp,
     question += ' [y/N]: '
 
   var = environ_cp.get(var_name)
+  if var is not None:
+    _var_content = var.strip().lower()
+    _true_contents = ['1', 'y', 'yes', 'true']
```

Comment:
You can use tuples instead of lists, to avoid inadvertent changes the them.

Reply:
Done. Thanks